drivers/rawexec/driver.go

```diff
@@ -107,6 +111,10 @@ var (
 	}
 )
 
+type UserIDValidator interface {
+	HasValidIDs(user *user.User) error
+}
```

Comment:
We've defined this for both drivers, so maybe this interface type should live in the validators package?

Reply:
They might grow in different directions, I don't want to limit it from the start

Reply:
I think I see where you're going with that, inasmuch as many each driver only cares about a subset of the methods on the object returned by `validator.NewValidator`.